packages/boba/gateway/src/components/listFarm/listFarm.js

```diff
@@ -298,20 +298,20 @@ class ListFarm extends React.Component {
               </Typography>
             </S.GridItemTag>
 
-            <S.GridItemTag item 
-              xs={4} 
+            <S.GridItemTag item
+              xs={4}
               md={1}
               >
               {isMobile ? (
                 <Typography variant="overline" sx={{opacity: 0.7, paddingRight: '5px'}}>Earned</Typography>
               ) : (null)}
               <Typography variant="body1">
-                {userReward ? `${logAmount(userReward, decimals, 2)}` : `0`}
+                {userReward ? `${logAmount(userReward, decimals, 5)}` : `0`}
```

Comment:
should we display this on the dropdown box amount as well?( L337)

Reply:
Makes sense on mobile yeah!

Reply:
Done!